Podfile

```diff
@@ -206,7 +206,7 @@ abstract_target 'Apps' do
 
   # Production
 
-  pod 'Automattic-Tracks-iOS', '~> 0.13.0-beta.2'
+  pod 'Automattic-Tracks-iOS', '~> 0.13'
```

Comment:
Just in case we'll end up in this PR to check which pods changed in 20.9, I wanted to note down that `WordPressAuthenticator` changed as well but that I made the change in https://github.com/wordpress-mobile/WordPress-iOS/commit/016679ca24bdff97e94e31f6fa66e1a88f87143e as part of unblocking the merge of a bug fix PR before starting the code freeze.